crates/continuations/src/lib.rs

```diff
@@ -87,11 +88,11 @@ pub struct Payloads {
 }
 
 impl Payloads {
-    pub fn new(capacity: usize) -> Self {
+    pub fn new(capacity: u32) -> Self {
         let data = if capacity == 0 {
             ptr::null_mut()
         } else {
-            let mut args = Vec::with_capacity(capacity);
+            let mut args = Vec::with_capacity(capacity as usize);
             let args_ptr = args.as_mut_ptr();
             args.leak();
```

Comment:
I think this is redundant. The previous line should already ensure that the lifetime of `args` is untracked.